pkg/trace/api/profiles.go

```diff
@@ -22,9 +22,9 @@ import (
 
 const (
 	// profilingURLTemplate specifies the template for obtaining the profiling URL along with the site.
-	profilingURLTemplate = "https://intake.profile.%s/v1/input"
+	profilingURLTemplate = "https://intake.profile.%s/api/v2/profile"
```

Comment:
❓ We should probably make the same change in [dd-trace-go](https://github.com/DataDog/dd-trace-go/blob/v1/profiler/options.go#L55) itself for agentless uploads, right?

Reply:
the whole premise for deprecating `/v1/input` rather than us proxying payloads from that endpoint to v2 indefinitely is problematic, but agreed

Reply:
Created PROF-3915 and assigned it to myself for the next sprint.